### PR TITLE
fixed style differences

### PR DESCRIPTION
**Pull request by pablo-clueless**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-  --color-yellow: #ffea00;
-  --color-white: #ffffff;
+  --color-primary: #081F4A;
+  --color-secondary: #C5D7F9;
+  --color-secondary-light: #ECF2FD;
+  --color-text-secondary: #5B6773;
+  --color-yellow: #FFEA00;
+  --color-white: #FFFFFF;
   --live-bg: rgba(236, 242, 253, 0.6);
+  --body-font: 'Lato', sans-serif;
```
Comment (taiwoAkindele) [inline src/index.css:11]: Font is Inter

**Review by taiwoAkindele — APPROVED** [review]

File: src/index.css (hunk 1 above)
Comment (taiwoAkindele) [reply src/index.css:11]: You also have conflicts on your branch

Comment (pablo-clueless) [reply src/index.css:11]: > Font is Inter

The conflict is coming from the previous design, Lato was used.

Comment (pablo-clueless) [reply src/index.css:11]: > You also have conflicts on your branch

fixed.

Comment (PhilemonBrain) [reply src/index.css:11]: use inter please, thats what the designer specified

Comment (pablo-clueless) [reply src/index.css]: > use inter please, thats what the designer specified

Done.

**Pull request merged**